src/main/java/com/google/cloud/tools/maven/CloudSdkAppEngineFactory.java

```diff
@@ -102,12 +102,17 @@ public GenRepoInfoFile genRepoInfoFile() {
   }
 
   protected CloudSdk.Builder defaultCloudSdkBuilder() {
+    if (mojo.getcloudSdkPath() == null) {
+      mojo.downloadCloudSdk();
+    } else if (mojo.getCloudSdkVersion() != null) {
+      mojo.checkCloudSdk();
+    }
 
     ProcessOutputLineListener lineListener = new DefaultProcessOutputLineListener(mojo.getLog());
 
     return cloudSdkFactory
         .cloudSdkBuilder()
-        .sdkPath(mojo.getCloudSdkPath())
+        .sdkPath(mojo.getcloudSdkPath())
```

Comment:
Can't find where this uses the downloaded SDK's home path?

Reply:
I think I can fix this in the next PR; it'll be easier to do when I have a ManagedCloudSdk I can get the downloaded path from.

Reply:
Submitted an issue for this so I don't forget (#256)